### PR TITLE
Feature/bi 1502

### PR DESCRIPTION
**Pull request by cvacalares**
# Description
BI-1502 - Shared Ontology testing

# Dependencies
None

# Testing
[_Please include a link to a successful run of TAF for this change_](https://github.com/Breeding-Insight/taf/actions/runs/3071844282)

# Checklist:

- [x ] I have performed a self-review of my own code
- [ ] I have tested my code and ensured it meets the acceptance criteria of the story
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to documentation

### CODE REVIEW

**Review by nickpalladino — APPROVED** [review]

**Review by timparsons — CHANGES_REQUESTED** [review]
A few small readability and future-proofing suggestions

File: src/step_definitions/programManagementSteps.js [hunk 1]
```diff
@@ -613,6 +620,151 @@ Then(
   }
 );
 
+When(
+  /^user selects "([^"]*)" tab on Program Management page$/,
+  async function (args1) {
+    await page.section.programManagement.click("@configurationLink");
```
[the remark this answers is not in the document]
Comment (cvacalares) [reply src/step_definitions/programManagementSteps.js:626]: Done

File: src/step_definitions/programManagementSteps.js [hunk 2]
```diff
@@ -613,6 +620,151 @@ Then(
...
+
+Then(
+  /^user can see Configuration on Program Management page$/,
+  async function () {
+    await page.expect.section("@configurationForm").visible;
+  }
+);
+
+Then(
+  /^user can see "([^"]*)" section on Program Management page$/,
```
[the remark this answers is not in the document]
Comment (cvacalares) [reply src/step_definitions/programManagementSteps.js:638]: Done

File: src/step_definitions/programManagementSteps.js [hunk 3]
```diff
@@ -613,6 +620,151 @@ Then(
...
+  async function (args1) {
+    await page.section.configurationForm.assert.visible(
```
[the remark this answers is not in the document]
Comment (cvacalares) [reply src/step_definitions/programManagementSteps.js:640]: Done

File: src/step_definitions/programManagementSteps.js [hunk 4]
```diff
@@ -613,6 +620,151 @@ Then(
...
+      "@sharedOntologySection"
+    );
+  }
+);
+
+Then(
+  /^user can see "([^"]*)" message on Program Management page$/,
+  async function (args1) {
+    await page.section.configurationForm.assert.containsText(
```
[the remark this answers is not in the document]
Comment (cvacalares) [reply src/step_definitions/programManagementSteps.js:649]: Done

**Comment by cvacalares** [comment]
Test run of changes at https://github.com/Breeding-Insight/taf/actions/runs/3171605326.
Please review.
@timparsons

**Review by timparsons — APPROVED** [review]

**Pull request merged**